public_html/wp-content/plugins/wordcamp-speaker-feedback/wordcamp-speaker-feedback.php

```diff
@@ -19,13 +19,17 @@
 define( __NAMESPACE__ . '\OPTION_KEY', 'sft_feedback_page' );
 define( __NAMESPACE__ . '\QUERY_VAR', 'sft_feedback' );
 
-register_activation_hook( __FILE__, __NAMESPACE__ . '\activate' );
-register_deactivation_hook( __FILE__, __NAMESPACE__ . '\deactivate' );
-add_action( 'plugins_loaded', __NAMESPACE__ . '\load' );
-add_action( 'init', __NAMESPACE__ . '\add_page_endpoint' );
+// Only add actions to sites without the skip flag, and only if WC Post Types exist.
+if ( ! wcorg_skip_feature('speaker_feedback' ) && class_exists( 'WordCamp_Post_Types_Plugin' ) ) {
```

Comment:
```suggestion
if ( ! wcorg_skip_feature( 'speaker_feedback' ) && class_exists( 'WordCamp_Post_Types_Plugin' ) ) {
```

Weird that phpcs isn't catching this.